addons/binding/org.openhab.binding.denonmarantz/README.md

```diff
@@ -58,27 +58,26 @@ The DenonMarantz AVR supports the following channels (some channels are model sp
 
 | Channel Type ID         | Item Type    | Description  |
 |-------------------------|--------------|--------------|
-| General
+| *General*
```

Comment:
Please note that this table looks broken, I would expect 3 columns as well here.

Reply:
O is it, thought it was valid syntax to have a section break in the table.

Reply:
Maybe a feature unknown to me :-)